test/parallel/test-console.js

```diff
@@ -76,17 +102,33 @@ assert.equal('foo bar\n', strings.shift());
 assert.equal('foo bar hop\n', strings.shift());
```

Comment:
Sure, the first bunch of these can work that way since they're just testing equality. I'll do that.

For the ones doing `indexOf`, regex or other more complex tests, I think they're going to have to be on their own still. Some comments might be able to clarify them though.